_includes/views-vues/assurance-level-requirement/assurance-level-resultats.html

```diff
@@ -0,0 +1,215 @@
+  <!-- Harm Strings -->
+<td class="table-question-{{ question }}-harm">
+  {% for level in (1..4) %}{%
+    assign harmString = questionNumber | append: "-hint-a" | append: level %}{%
+    assign harmString = page.questions[harmString] | remove_first: "• " | replace: " • ", "</li><li>" | prepend: "<ul><li>" | append: "</li></ul>" }}
+  %}<div id="table-question-{{ question }}-harm-{{ level }}" class="hidden">{{ harmString }}</div>{%
+  endfor %}
+  <div id="table-question-{{ question }}-harm-NA" class="hidden"></div>
+</td>
+  <!-- Rationale -->
+<td><div id="table-question-{{ question }}-rationale"></div></td>
+</tr>{%
+endfor %}
+
+<!-- Summary row -->
+<tr style="border: solid; border-width: 2px; font-size: 30px;">
+<th id="header-summary">{{ page.overallAssessment }}</th>
+<td headers="header-summary" id="r{{ rowNumber }}h2" style="text-align: center; font-size: 30px;" class="wb-calculate" data-wb-calculate='{ "onInit": true, "eventTrigger": "removeClass.action.wb-fieldflow", "operations": [
+    { "type": "max", "query": ".assurance-level-element:not(.hidden), #min-assurance-level", "outputTarget": "#max-assurance-level-used" }]}'>
+  <div id="min-assurance-level" class="hidden">1</div>
+  <div id="max-assurance-level-used">1</div>
+</td>
+<td headers="header-summary" colspan="3" style="font-size: 20px;">
+  {% for level in (1..4) %}{%
+    assign assuranceLevel = level | downcase %}
+    <div id="assurance-{{ level }}-table" class="wb-calculate {% if level > 1 %} hidden {% endif %}" data-wb-calculate='{ "eventTrigger": "removeClass.action.wb-fieldflow", "operations": [
+          { "type": "conditional", "inputs": [
+            { "type": "=", "inputs": [
+              { "type": "number", "query": "#max-assurance-level-used" },
+              {{ level }}
+            ] }
+        ], "actionsTrue": [
+          { "type": "removeClass", "outputTarget": "#assurance-{{ level }}-table", "class": "hidden" }
+        ], "actionsFalse": [
+          { "type": "addClass", "outputTarget": "#assurance-{{ level }}-table", "class": "hidden" }
+        ] }
+      ] }'>
+     {{ page.assurance[ assuranceLevel ] }}
+    </div>{%
+  endfor %}
+  <div id="assurance-NA-table" class="hidden"></div>
+</td>
+</tr>
+<tr>
+  <td colspan="5" style="text-align: center"> {{ page.version }} </td>
+</tr>
+</tbody>
+</table>
+</div>
+
+<button type="button" class="btn btn-default" onclick="javascript:PrintElem($( '#table-results-div' ), 'LOA Assessment - ' + $('#table-detail-3').text() + ' - ' + $('#table-detail-2').text());">Print / Download PDF</button>
+<button type="button" class="btn btn-default wb-format-gen" data-wb-format-gen='{ "type": "csv", "rowSelector": "tr:not(.hidden)", "colSelector": "th, td div:not(.hidden), .non-scoring-result td", "container": "#table-results-div", "filename": "assurance-level-assessment" }'>{{ site.DownloadCSV[page.lang] }}</button>
+
+<script language="javascript" type="text/javascript">
```

Comment:
Custom JS should get pulled out to assets folder like the other files

Reply:
script has been pulled to assets